rpm/package.sh

```diff
@@ -79,25 +79,33 @@ function download_flume(){
 
 function copy_cygnus_startup_script(){
     _logStage "######## Copying the cygnus startup script into the apache-flume... ########"
-    rm ${RPM_PRODUCT_SOURCE_DIR}/bin/flume-ng
-    cp $BASE_DIR/target/classes/cygnus-flume-ng ${RPM_PRODUCT_SOURCE_DIR}/bin
+    rm -rf ${RPM_PRODUCT_SOURCE_DIR}/bin/cygnus-flume-ng
```

Comment:
It is removed from the target rpm directory (`${RPM_PRODUCT_SOURCE_DIR}/bin/cygnus-flume-ng`) and copy again from sources directory (`${BASE_DIR}/cygnus-common/src/main/resources`). This is to have the latest version of this script.